app/controllers/api/v0/admin/configs_controller.rb

```diff
@@ -2,12 +2,26 @@ module Api
   module V0
     module Admin
       class ConfigsController < ApiController
+        include SiteConfigParams
+
         before_action :authenticate_with_api_key_or_current_user!
         before_action :authorize_super_admin
 
         def show
           @site_configs = SiteConfig.all
         end
+
+        def update
```

Comment:
this needs

```ruby
skip_before_action :verify_authenticity_token, only: %i[update]
```

at the top.

If you test it manually it won't work because it checks for a CSRF token

Reply:
👍